code401challenges/src/test/java/code401challenges/LinkedListTest.java

```diff
@@ -0,0 +1,72 @@
+package code401challenges;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+    LinkedList randomList = new LinkedList();
+    @Test
+    public void insertTest(){
+
+        assertEquals(
+                "This should give back 5.",
+                5,
+                randomList.insert(5).value
```

Comment:
This is testing that you are returning a Node w/ the correct value, but not that it's actually in the LinkedList or in the place you are expecting.